Processes/How-To-Submit-Content-to-the-Docs-Portal.md

```diff
@@ -0,0 +1,26 @@
+## How To Submit Content to EOS Documentation Portal
+
+**Scenario**: You have written content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com) in .md format. Now what do you need to do?
+
+Complete the following steps to submit your content to DevRel:
+
+1. Create a PR in the DevRel repository
```

Comment:
External contributors don't have the right to create PRs directly in the ENF repositories. This item should read:

1\. Create a PR from a fork.

Reply:
Done